microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -596,6 +596,26 @@ Microsoft also sends the product price, product image, product name, ratings, an
 
 The Microsoft Edge shopping feature requires sharing of cookie information with Bing.  For example, the cookies may be used for debugging, fraud detection, and analytics.  The Microsoft shopping service returns prices from other retailers, historical price trends, and any available coupons for that website.
 
+![Shopping and coupons](./media/inboxcoupons.png)
+
+To help you find and keep track of coupons in your inbox, Microsoft Edge may optionally scan your connected email account for coupons in promotional emails. The Microsoft shopping service extracts coupons from promotional emails sent by supported retailers. The extracted coupons are stored and managed by the Microsoft shopping service and is accessible to you. Inbox coupons will not appear when using **Guest mode** or **InPrivate**.
+
+![Shopping and coupons](./media/inboxcoupons-settings.png)
+
+To control which retailers' emails get scanned by Microsoft Edge, complete the following actions: 
+
+1.  Click on the **“Your Shopping”** tab on the shopping sidebar 
+1.  Go to the Inbox coupons card and click on the “...” button 
+1.  Click on the **“Settings”** link and view retailers in your inbox 
+1.  Control retailer email preferences via toggles 
```

Comment:
```suggestion
To control which retailers' emails are scanned by Microsoft Edge:

1.  On the **Shopping** sidebar, click the **Your Shopping** tab.

1.  In the **Inbox coupons** card, click the **Settings** (**...**) button:

    ![Shopping and coupons](./media/inboxcoupons-settings.png)

1.  In the **Retailers in your inbox** section, expand the section for a retailer, and then turn the toggle on or off.
```

Reply:
i haven't reproduced the actual UI yet; i can't actually follow the draft steps or revised steps.  I'm not logged in, and there is no "Inbox coupons" card.

Reply:
@mikehoffms Inbox coupons is 100% in US market and we are flighting in UK/AU by end of May and will start our flights to DE/FR in June. If you are having trouble viewing our experience in the US please let me know.

To access the experience outside the US you will need to use Edge Launcher and run our flag "--enable-features=msShoppingExp52"

![image](https://github.com/MicrosoftDocs/edge-developer/assets/133438973/1adbb69c-5257-4fdb-a3af-d6ae84098252)